src/store/reducers/loader.js

```diff
@@ -0,0 +1,27 @@
+import * as actionTypes from "../actionTypes";
+
+export const INITIAL_STATE = false;
+
+export default function loaderReducer(state = INITIAL_STATE, action) {
+  switch (action.type) {
+    case actionTypes.FETCH_USER_SONGS_START:
+    case actionTypes.FETCH_USER_SONGS_FAIL:
+    case actionTypes.ADD_ALBUMS_START:
+    case actionTypes.ADD_ALBUMS_FAIL:
+    case actionTypes.ADD_ARTISTS_START:
+    case actionTypes.ADD_ARTISTS_FAIL:
+    case actionTypes.FETCH_ALBUM_SONGS_START:
+    case actionTypes.FETCH_ALBUM_SONGS_FAIL:
+    case actionTypes.FETCH_ARTISTS_SONGS_START:
+    case actionTypes.FETCH_ARTISTS_SONGS_FAIL:
```

Comment:
actually we need to hide loader when FAIL action happens

Reply:
Fixed